unified-codes/libs/ui/src/lib/components/inputs/organisms/LoginForm.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
 
+import { IUserCredentials } from '@unified-codes/data';
+
 import Button from '../atoms/Button';
 import Grid from '../../layout/atoms/Grid';
 import LoginInput from '../molecules/LoginInput';
 import Typography from '../../data/atoms/Typography';
 
 export interface LoginFormProps {
-  onSubmit?: (username: string, password: string) => void;
+  onSubmit?: (IUserCredentials) => void;
```

Comment:
I'm not sure if this is just my dev environment - typescript didn't like this for me and suggested the following change which allowed me to build web:
`onSubmit?: (arg0: IUserCredentials) => void;`

Reply:
Ah yup, that should be `credentials: IUserCredentials`. Will fix!

Reply:
9bca63665f07d2bddfb19cb2696a9f064b689bee.